Content.Server/Chemistry/Components/SolutionRegenerationComponent.cs

```diff
@@ -14,31 +14,31 @@ public sealed partial class SolutionRegenerationComponent : Component
     /// <summary>
     /// The name of the solution to add to.
     /// </summary>
-    [DataField("solution", required: true), ViewVariables(VVAccess.ReadWrite)]
+    [DataField("solution", required: true)]
     public string SolutionName = string.Empty;
 
     /// <summary>
     /// The solution to add reagents to.
     /// </summary>
-    [DataField("solutionRef")]
-    public Entity<SolutionComponent>? Solution = null;
+    [DataField]
+    public Entity<SolutionComponent>? SolutionRef = null;
```

Comment:
Didn't Sloth say to convert this to an EntityUid? 🤔

Reply:
Yes, but I haven't figured out how to do that yet.

I don't remember exactly what issue I ran into, but I think it was that almost nothing else expects solutions to be entities themselves, so it was going to need a massive overhaul. But that could be my foggy memory.

Reply:
yeah, it would kinda involve fetching the comp everywhere it's used...

Reply:
it's already broken saving by whoever merged + coded it so eh, not making the problem worse.